content/en/project/opimon/index.md

```diff
@@ -0,0 +1,59 @@
+---
+# Documentation: https://sourcethemes.com/academic/docs/managing-content/
+
+title: "An Interactive Monitoring Tool for OpenFlow Networks (Opimon)"
+summary: ""
+authors: [Wassapon Watanakeesuntorn]
+tags: ["SDN"]
+categories: []
+date: 2020-10-05T14:00:24+09:00
+
+# Optional external URL for project (replaces project detail page).
+external_link: ""
+
+# Featured image
+# To use, add an image named `featured.jpg/png` to your page's folder.
+# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
+image:
+  caption: ""
```

Comment:
Please rename `opimon.png` to `featured.png` so that a thumbnail image will show up in [the list of projects](https://deploy-preview-60--sdlab.netlify.app/en/research/). The caption text can be set in this line.